src/goto-programs/abstract-interpretation/interval_domain.cpp

```diff
@@ -910,6 +937,63 @@ void interval_domaint::transform(
 
   default:;
```

Comment:
Maybe remove this default label, so the compiler can help catching missing cases?

Reply:
I added the the cases and removed the default. Maybe the try/catch could be the next step!